source/sdk/math.ooc

```diff
@@ -306,7 +299,7 @@ extend Float {
 		value
 	}
 	minusPiToPiOverTwo: static func (value: This) -> This {
```

Comment:
Again, not on you, but if we really should have this method it should probably be called `minusPiToPiHalf` and not be static, which is true for the methods above as well.